emissionsapi/preprocess.py

```diff
@@ -156,9 +142,7 @@ def write_to_database(session, data):
 
             # Check if any of the data objects are set to NotANumber with
             # filter_data() to skip writing them into the database
-            if (not(numpy.isnan(data.longitude[i, j]) or
-                    numpy.isnan(data.latitude[i, j]) or
-                    numpy.isnan(data.data[i, j]))):
+            if (numpy.isfinite(data.data[i, j])):
```

Comment:
why choosing isfinite over isnan?

Reply:
The main reason is simply that I wanted to avoid the negation since that is always easier to read. `isfinite()` is almost a negation except that now `INF` values are filtered as well as `NaN` values. I think this makes sense since an infinite measurement should not exist and clearly marks a bad value as well.